docs/getting-started/beginner-guide.md

```diff
@@ -1,85 +1,102 @@
+* [Decred.org blog](https://blog.decred.org)
+* [Decred Medium](https://medium.com/decred) (curated content from community members)
+* [Press coverage/releases](https://decred.org/press/)
+* [The Decred Digest](https://thedecreddigest.com/) - community blog
 
-## Decrediton Path 
+**Video**
 
-The following guide will help you load Decrediton:
+* [Decred Youtube channel](https://www.youtube.com/decredchannel)
 
-* [Decrediton Setup](/getting-started/user-guides/decrediton-setup.md)
-* [Using Decrediton](/getting-started/user-guides/using-decrediton.md)
+## Community
 
-## Command-Line Path 
+In addition to the [Politeia proposals site](www.proposals.decred.org), the Decred community can be found on a [variety of platforms](https://decred.org/community/).
 
-The following guides, in order, will get you started with the Command-Line Applications (`dcrd`, `dcrwallet`, `dcrctl`):
+### Chat 
 
-* [Command-Line Differences](/getting-started/cli-differences.md)
-* [Startup Option Basics](/getting-started/startup-basics.md)
-* [dcrd Setup](/getting-started/user-guides/dcrd-setup.md)
-* [dcrwallet Setup](/getting-started/user-guides/dcrwallet-setup.md)
-* [dcrctl Basic Usage](/getting-started/user-guides/dcrctl-basics.md)
+The following chat platforms are bridged, such that one's messages are relayed to users on the other platforms:
 
-## General Guides 
+* [Slack](https://slack.decred.org/)
+* [Matrix (via Riot.im)](https://riot.im/app/#/room/#general:decred.org)
+* [Discord](https://discord.gg/GJ2GXfz)
+* [IRC](https://webchat.freenode.net/?channels=decred&uio=d4)
 
-The following guides are independent of the different applications:
+Telegram, Rocket.Chat and KakaoTalk are partially bridged.
 
-* [Using Testnet](/getting-started/using-testnet.md)
-* [Obtaining DCR](/getting-started/obtaining-dcr.md)
-* [Using the Block Explorer](/getting-started/using-the-block-explorer.md)
-* [Proof-of-Stake Guide](/mining/proof-of-stake.md)
-* [Mainnet Voting Guide](/getting-started/user-guides/agenda-voting.md)
+### Social Media
+
+* [Reddit](https://www.reddit.com/r/decred)
+
+* [Twitter]() (account operated by small group of contributors)
+* [Forum](https://forum.decred.org/)
+* [Facebook](https://facebook.com/decredproject) 
+
+### Code
+
+Code for a variety of software projects lives on [GitHub](https://github.com/decred). Issues can be used to report bugs or suggest features. Pull Requests can be issued to merge work into the main repository branch.
+
+Selected GitHub repositories:
+
+* [decrediton](https://github.com/decred/decrediton) - GUI wallet
+* [dcrd](https://github.com/decred/dcrd) and [dcrwallet](https://github.com/decred/dcrwallet) - Decred daemons
+* [politeia](https://github.com/decred/politeia) and [politeiagui](https://github.com/decred/politeiagui) - Politeia back and front end
+* [dcrdata](https://github.com/decred/dcrdata) - block explorer
+* [dcrdocs](https://github.com/decred/dcrdocs) - this [docs site](http://docs.decred.org)
```

Comment:
Can we update the dcrdocs link please from 
`[docs site](http://docs.decred.org)`
to
`[docs site](/)`

This will mean that the link will work correctly whether the site is running locally or on docs.decred.org.

Reply:
Also will preserve http/https